shoop/themes/classic_gray/plugins.py

```diff
@@ -84,3 +86,79 @@ def get_context_data(self, context):
             "count": count,
             "orderable_only": orderable_only,
         }
+
+
+class SocialMediaLinksPlugin(TemplatedPlugin):
+    """
+    An xtheme plugin for displaying site links to common social media sites.
+    """
+    identifier = "classic_gray.social_media"
+    name = _("Social Media Links")
+    template_name = "classic_gray/social_media_links.jinja"
+    editor_form_class = SocialMediaLinksPluginForm
+    fields = [
+        ("topic", TranslatableField(label=_("Topic"), required=False, initial="")),
+        ("text", TranslatableField(label=_("Title"), required=False, initial="")),
+        ("icon_size", forms.ChoiceField(label=_("Icon Size"), required=False, choices=[
+            ("", "Default"),
+            ("lg", "Large"),
+            ("2x", "2x"),
+            ("3x", "3x"),
+            ("4x", "4x"),
+            ("5x", "5x"),
+        ], initial="")),
+        ("alignment", forms.ChoiceField(label=_("Alignment"), required=False, choices=[
+            ("", "Default"),
+            ("left", "Left"),
+            ("center", "Center"),
+            ("right", "Right"),
+        ], initial="")),
+    ]
+
+    icon_classes = {
+        "Facebook": "facebook-square",
+        "Flickr": "flickr",
+        "Google Plus": "google-plus-square",
+        "Instagram": "instagram",
+        "Linkedin": "linkedin-square",
+        "Pinterest": "pinterest",
+        "Tumbler": "tumblr",
+        "Twitter": "twitter",
+        "Vimeo": "vimeo",
+        "Vine": "vine",
+        "Yelp": "yelp",
+        "Youtube": "youtube",
+    }
```

Comment:
I don't see why this should be a dict. IMHO list of tuples would be better. Also, since there is a need for translated string, add it too.
```python
icon_data = [
    # (name, icon_class, label)
    ("Facebook", "facebook-square", _("Facebook")),
    ("Flickr", "flickr", _("Flickr")),
    ...
```